### DISCUSSION TITLE
[FEATURE]现在docker拉取不下来，需要配置代理，为什么不把docker的镜像配置成阿里云公开的

### DISCUSSION THREAD

**Discussion by xiahao90** [1Panel-dev/1Panel#6244] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 请描述您的需求或者改进建议

docker正常拉取

### 请描述你建议的实现方案

现在docker拉取不下来，需要配置代理，为什么不把app里面的docker的镜像全部传到阿里云的镜像上面全，配置成阿里云公开的。在吧app的配置成阿里云的前缀不就方便了

**Comment by wanghe-fit2cloud** [comment]
目前的版本确实需要设置代理或添加镜像加速器，才能顺利拉取镜像。v2 版本将会支持自定义镜像仓库的功能。

单纯切换为阿里云仓库地址，许多应用镜像仍需要手动维护，目前我们没有足够精力处理这件事。因为大部分应用镜像都托管在 Docker Hub 或 GitHub 等仓库，应用发布新版本后的同步工作也同样需要大量精力。

**Reply by xiahao90** [reply]
我觉得精力应该还好，开发者提交app的时候都给了 Docker Hub上面的docker的名称，你们写一个脚本一键拉取再提交，再替换app配置就好了，目前阿里云的个人免费都有300个仓库，也是够用的。阿里云只是一个选项华为应该等也有相关的服务。因为怕开发者提交自己的仓库会搞一些自己的东西在docker里面，只能由你们同统一提交到你们官方仓库。

**Reply by xiahao90** [reply]
https://github.com/ikrong/sync-docker-image 这个项目可以快速将docker hub同步到阿里云，应该可以写个自动程序将我们使用的镜像同步到阿里云。可避免手动操作

**Reply by wanghe-fit2cloud** [reply]
目前 docker hub 国内已经可以正常访问，拉取镜像了。
> v2 版本会支持用户手动设置仓库地址。

**Reply by xiahao90** [reply]
又不能拉取了
